x-pack/platform/plugins/shared/inference/server/chat_complete/anonymization/anonymize_messages.ts

```diff
@@ -49,9 +48,9 @@ export async function anonymizeMessages({
   });
 
   const anonymizedMessages = messages.map((original, index) => {
-    const map = records[index];
+    const anonymizedRecord = records[index];
```

Comment:
nit: mind renaming `records` to `anonymizedRecords` for clarity?

```suggestion
    const anonymizedRecord = anonymizedRecords[index];
```